webrender/Cargo.toml

```diff
@@ -17,7 +17,7 @@ app_units = "0.4"
 bincode = "1.0.0-alpha6"
 bit-set = "0.4"
 byteorder = "1.0"
-euclid = "0.11.2"
+euclid = ">= 0.11, < 0.14"
```

Comment:
I'm not 100% sure about this being a non-breaking change. Is this approach established elsewhere for a reference?